langchain-core/src/runnables/remote.ts

```diff
@@ -214,11 +214,12 @@ function deserialize<RunOutput>(str: string): RunOutput {
   return revive(obj);
 }
 
-function removeCallbacks(
+function removeCallbacksAndSignal(
   options?: RunnableConfig
-): Omit<RunnableConfig, "callbacks"> {
+): Omit<RunnableConfig, "callbacks" | "signal"> {
```

Comment:
Do we have to handle `timeout` here too?

Reply:
No, timeout can safely be passed to JSON.stringify